sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -43,24 +43,30 @@ import org.apache.spark.tags.ExtendedSQLTest
  *
  * To run the entire test suite:
  * {{{
- *   build/sbt "sql/test-only *SQLQueryTestSuite"
+ *   build/sbt -Phive-1.2 "sql/test-only *SQLQueryTestSuite"
```

Comment:
First of all, we had better use `-Phive-2.3`.
Second, please wait for a while. This PR will become obsolete soon.
As you see, it's still under **slow** changing to avoid any breakage.
Yesterday, we added `hive-1.2/2.3`. Today, we regenerates the manifest via `test-dependencies.sh`.
And, I'm working on the dependency changes. The following is the history of the migration.

a1706e2fa7 [SPARK-30005][INFRA] Update `test-dependencies.sh` to check `hive-1.2/2.3` profile
a60da23d64 [SPARK-30007][INFRA] Publish snapshot/release artifacts with `-Phive-2.3` only
13338eaa95 [SPARK-29554][SQL][FOLLOWUP] Rename Version to SparkVersion
6625b69027 [SPARK-29981][BUILD][FOLLOWUP] Change hive.version.short
c98e5eb339 [SPARK-29981][BUILD] Add hive-1.2/2.3 profiles

Reply:
Thanks for  the instructions

Reply:
Thank you for understanding and sorry for the inconvenience. It's difficult to change the tires of the running cars.

Reply:
Never mind. I will leave this pull request open for a while in case someone encounters the same issue until it is completely fixed.

Reply:
Although I'm still working on, I create a WIP PR to share with you.
- https://github.com/apache/spark/pull/26658